src/uct/ib/base/ib_iface.h

```diff
@@ -58,6 +58,17 @@ enum {
 #endif
 };
 
+
+/**
+ * IB address packing flags
+ */
+enum {
+    UCT_IB_ADDRESS_PACK_FLAG_ETH           = UCS_BIT(0),
```

Comment:
there are other address flags in `ib_device.h`
Maybe reuse them?

Reply:
these are flags for the serialized address which is passed to remote side, we should not reuse them for as func params